README.md

```diff
@@ -223,29 +221,31 @@ await deleteFiles({
 });
 ```
 
-### @huggingface/agents example
+### @huggingface/mcp-client example
 
 ```ts
-import { HfAgent, LLMFromHub, defaultTools } from '@huggingface/agents';
+import { Agent } from '@huggingface/mcp-client';
 
 const HF_TOKEN = "hf_...";
 
-const agent = new HfAgent(
-  HF_TOKEN,
-  LLMFromHub(HF_TOKEN),
-  [...defaultTools]
-);
-
+const agent = new Agent({
+  provider: "auto",
+  model: "Qwen/Qwen2.5-72B-Instruct",
+  apiKey: HF_TOKEN,
+  servers: [
+    {
+      // Playwright MCP
+      command: "npx",
+      args: ["@playwright/mcp@latest"],
+    },
+  ],
+});
 
-// you can generate the code, inspect it and then run it
-const code = await agent.generateCode("Draw a picture of a cat wearing a top hat. Then caption the picture and read it out loud.");
-console.log(code);
-const messages = await agent.evaluateCode(code)
-console.log(messages); // contains the data
 
-// or you can run the code directly, however you can't check that the code is safe to execute this way, use at your own risk.
-const messages = await agent.run("Draw a picture of a cat wearing a top hat. Then caption the picture and read it out loud.")
-console.log(messages);
+await agent.loadTools();
+for await (const chunk of agent.run("Draw a picture of a cat wearing a top hat. Then caption the picture and read it out loud.")) {
+  console.log(chunk);
+}
```

Comment:
I'd update the prompt for something `@playwright/mcp@latest` is able to run + showcase how to read Agent's output:

```suggestion
for await (const chunk of agent.run("What are the top 5 trending models on Hugging Face?")) {
    if ("choices" in chunk) {
        const delta = chunk.choices[0]?.delta;
        if (delta.content) {
            console.log(delta.content);
        }
    }
}
```

Reply:
sounds good, thanks!